### PR TITLE
Remove translations with duplicate values

### PR DESCRIPTION
**Pull request by dkanada**
```
OptionPlayed
OptionUnplayed
OptionFavorite
OptionContinuing
OptionEnded
OptionHasSubtitles
OptionHasTrailer
OptionDownloadArtImage
OptionDownloadBackImage
OptionDownloadBannerImage
OptionDownloadBoxImage
OptionDownloadDiscImage
ButtonRestart
OptionProfileAudio
TabSettings
OptionAuto
OptionHomeVideos
ButtonSettings
TabProfile
OptionBlockBooks
OptionBlockMovies
OptionBlockMusic
ButtonResetPassword
OptionRuntime
OptionAlbumArtist
OptionAlbum
OptionArtist
HeaderSettings
ButtonProfile
OptionAscending
OptionDescending
ButtonSelectServer
ButtonSettings
ButtonGuide
OptionHasSpecialFeatures
OptionDownloadPrimaryImage
OptionDownloadLogoImage
OptionDownloadThumbImage
OptionDownloadMenuImage
OptionNone
OptionBlockChannelContent
OptionBlockLiveTvChannels
OptionBlockTrailers
OptionBlockTvShows
HeaderSpecialFeatures
HeaderMusicVideos
LabelVideo
OptionNameSort
```

### CODE REVIEW

File: src/components/notifications/notifications.js [hunk 1]
```diff
@@ -248,7 +248,7 @@ events.on(serverNotifications, 'RestartRequired', function (e, apiClient) {
         [
             {
                 action: 'restart',
-                title: globalize.translate('ButtonRestart'),
+                title: globalize.translate('HeaderRestart'),
```
Comment (EraYaN) [inline src/components/notifications/notifications.js:251]: Above the one that was picked was the Button* prefix, and here it's the Header* prefix. Maybe the Button prefix is more general.

**Comment by EraYaN** [comment]
This does seem to introduce a bunch of in consistencies, with "neighboring" translations. Like some lists now have three with some prefix and then one without (for example the "Audio" one).

File: src/components/notifications/notifications.js (hunk 1 above)
Comment (dkanada) [reply src/components/notifications/notifications.js]: fixed

**Comment by dkanada** [comment]
The only items left with dynamic translations are included in the list below. I'm also including the favorite headers because I think the prefix is redundant.

```
ColorPrimaries
ColorSpace
ColorTransfer
HeaderFavoriteBooks
HeaderFavoriteMovies
HeaderFavoriteShows
HeaderFavoriteEpisodes
HeaderFavoriteAlbums
HeaderFavoritePeople
HeaderFavoriteArtists
HeaderFavoriteSongs
HeaderFavoriteVideos
HeaderFavoritePlaylists
MediaInfoStreamTypeAudio
MediaInfoStreamTypeData
MediaInfoStreamTypeEmbeddedImage
MediaInfoStreamTypeSubtitle
MediaInfoStreamTypeVideo
OptionBanner
OptionList
OptionPoster
OptionPosterCard
OptionThumb
OptionThumbCard
PlaybackErrorNoCompatibleStream
```

**Review by MrTimscampi — APPROVED** [review]

**Review by EraYaN — APPROVED** [review]

**Pull request merged**